dlrover/go/operator/pkg/controllers/master/master.go

```diff
@@ -238,6 +238,16 @@ func NewMasterTemplateToJob(job *elasticv1alpha1.ElasticJob, masterImage string)
 		" --platform pyk8s --namespace %s --job_name %s --port %d",
 		job.Namespace, job.Name, masterServicePort,
 	)
+
+	// for extra arguments
+	var supportedArguments = []string{"pending_timeout", "pending_fail_strategy"}
+	for _, item := range supportedArguments {
+		var value, exists = job.Annotations[item]
```

Comment:
```
if value, ok := job.Annotations[item]; ok {
    command += fmt.Sprintf(" --%s %s", item, value)
}
```

Another proposal, we can enable setting the master pod spec in the elasticjob like a worker pod spec. Then, the user can configure the command and resource of the master pod.

Reply:
> Another proposal, we can enable setting the master pod spec in the elasticjob like a worker pod spec. Then, the user can configure the command and resource of the master pod.

Issue created: https://github.com/intelligent-machine-learning/dlrover/issues/1432

Reply:
done